sympy/geometry/ellipse.py

```diff
@@ -140,11 +140,14 @@ def __new__(
                 "eccentricity" must not be None.'''))
 
         if eccentricity is not None:
-            if hradius is None:
+            if eccentricity.is_real == True and eccentricity > 1:
```

Comment:
```suggestion
            if eccentricity.is_real is True and eccentricity >= 1:
```
AFAIK, if eccentricity is 1 then it should be a parabola.

Reply:
Added